addon/components/polaris-action-list/section.js

```diff
@@ -1,13 +1,12 @@
+import { tagName, layout as templateLayout } from '@ember-decorators/component';
+import { action, computed } from '@ember/object';
```

Comment:
Just wondering if it's possible to use `action as actionDecorator` (or something better-named 😅) to avoid having to rename every property that's called `action` - that way we might be able to avoid diverging from the React implementation's interface and eliminate the need for breaking changes... 🙏

Reply:
do we have many properties called `action`?

PS: I don't think we should make this decision based on whether we need to make a breaking change or not (that's why semver exists) or just to avoid diverging from React implementation - we already diverge in quite a few places. From my view, we should think about what would be best for the future of the add-on

Reply:
>do we have many properties called `action`?

Not sure without checking tbh 🤷‍♂ 

I agree with the rest of what you're saying... but if there's a way to have both (i.e. use the `@action` decorator _and_ keep our existing naming) just by choosing a different name for the decorator, why would we choose a breaking change over an internal rename? 🤔

Reply:
> but if there's a way to have both (i.e. use the @action decorator and keep our existing naming) just by choosing a different name for the decorator, why would we choose a breaking change over an internal rename? 

not sure....kind of leaning towards ensuring we avoid risks of conflicting with Ember internals out-of-the-box in detriment of not matching React implementation 💯(we're already doing this in a few places, right?) - since while that's nice to have, not really a requirement....but I'm not super strong on this